### PR TITLE
Certified Copy Status

### PR DESCRIPTION
**Pull request by jgbenito7**
This PR affects the download links on the receipt.

Shows "Awaiting Clerk Approval" if the certified copy is `pending`. Shows "Certified Copy Rejected" if the certified copy status is `rejected`. If the certified copy is ready, the user will receive an email. At that point they can view the receipt to download.

Closes
- https://kofile.tpondemand.com/entity/50161-ui-in-receipt-view-both-order
- https://kofile.tpondemand.com/entity/50160-ui-in-receipt-view-both-order

### CODE REVIEW

File: src/DownloadLink.tsx [hunk 1]
```diff
@@ -0,0 +1,113 @@
...
+    <div className={style.downloadWrap}>
+      <span className={style.linkError}>Certified Copy Rejected</span>
+    </div>
+  );
+
+  const error = (
+    <span className={style.linkError}>{props.downloadLink.error}</span>
+  );
+
+  if (certificationStatus === CertificationStatuses.PENDING) {
```
Comment (EthanJStark) [inline src/DownloadLink.tsx]: My work in [SOS](https://github.com/kofile/search-order-service/pull/255) only appends a `certificationStatus: "approved"` key:value to an Order.item when it's PATCHed to approved. `CertificationRequest` table has the full range of `CertificationStatuses`, but I think that React Order Receipt generally sources data from Order.items, not CertificationRequest. Do we need a mechanism to get a PENDING status into this receipt?

Comment (jgbenito7) [reply src/DownloadLink.tsx]: On the UI we will need to pass that in

Comment (EthanJStark) [reply src/DownloadLink.tsx]: Ok, amending my PR to update Order.items in the cases of status: "pending" and status: "rejected"

Comment (jgbenito7) [reply src/DownloadLink.tsx]: oh i see what you mean now.

Comment (EthanJStark) [reply src/DownloadLink.tsx]: [My PR](https://github.com/kofile/search-order-service/pull/255) now handles all state changes; just waiting on reviews.

**Pull request merged**